packages/vaadin-lumo-styles/mixins/required-field.js

```diff
@@ -40,6 +40,10 @@ const requiredField = css`
     padding-top: var(--lumo-space-m);
   }
 
+  :host([has-label]) ::slotted([slot='tooltip']) {
+    --vaadin-tooltip-offset-bottom: calc((var(--lumo-space-m) - var(--lumo-space-xs)) * -1);
```

Comment:
For some reason this happens: the tooltip is not shown if the field is entered from above / over the label.

https://user-images.githubusercontent.com/1222264/190110041-064800ba-5f85-45cb-8b64-f02d759849e8.mp4

Reply:
This happens because `mouseleave` event is fired as mouse moves to immediately open `vaadin-tooltip-overlay`.
I will submit a separate PR to fix this, as it's likely to also happen for different position / offset combinations.